downloader/src/funTest/kotlin/vcs/CvsWorkingTreeTest.kt

```diff
@@ -19,19 +19,20 @@
 
 package org.ossreviewtoolkit.downloader.vcs
 
-import org.ossreviewtoolkit.model.VcsType
-import org.ossreviewtoolkit.utils.ORT_NAME
-import org.ossreviewtoolkit.utils.getOrtDataDirectory
-import org.ossreviewtoolkit.utils.safeDeleteRecursively
-import org.ossreviewtoolkit.utils.unpack
-
 import io.kotest.core.spec.Spec
+import io.kotest.core.spec.style.StringSpec
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
-import io.kotest.core.spec.style.StringSpec
 
 import java.io.File
 
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.utils.Ci
+import org.ossreviewtoolkit.utils.ORT_NAME
+import org.ossreviewtoolkit.utils.getOrtDataDirectory
+import org.ossreviewtoolkit.utils.safeDeleteRecursively
+import org.ossreviewtoolkit.utils.unpack
+
```

Comment:
Just for my information: What import ordering are we using actually? Is it two sections like:

1. All non ORT imports sorted
2. All ORT imports sorted

Reply:
IMO the import ordering has always been like this (until the recent renaming of ORT packages):

1. Sort all imports alphabetically
2. Separate all top-level imports by a blank line (i.e. separate `com.` from `org.` imports)
3. Additionally also add a blank line between `com.` and `org.` imports that differ at the second level (i.e. separate `org.apache` from `org.ossreviewtoolkit` imports)

I'm still planning to automate checks / auto-formatting this, and beginnings are in the `ort-import-order` branch, but this is low-priority work for me and progresses slowly.

Reply:
Import ordering still needs to be fixed after renaming the package from `com.here.ort` to `org.ossreviewtoolkit`. This will be done with a script eventually. But the pattern is still alphabetical with blank lines between different TLDs and bonus blank lines around the ORT package.